gcovr/templates/source_page.html

```diff
@@ -147,6 +158,25 @@
         {% endif %}
       </td>
       {% endif %}
+      {% if SHOW_CALL %}
+      <td class="linecall">
+        {% if row.linecall %}
+          <details class="linebranchDetails">
```

Comment:
Add a separate CSS style `linecallDetails` here to be able to format it separately.
The same for the other styles.

Reply:
Used new variables for call coverage and created new styles for css elements at
https://github.com/gcovr/gcovr/commit/7d45c6fd01f1e1e703cf0d7e15039308c3bf9a1d